templates/mycomics/bits/single_collection.html

```diff
@@ -20,8 +20,7 @@ <h3 style="display: inline-block;">{{ collection }} ({{ collection.items.count }
       </button>
       {% if not collection|is_default_collection %}
       <button type="button" class="btn btn-primary btn-sm"
-              style="display: inline-block;" data-toggle="modal"
```

Comment:
that is not needed ?

Reply:
The ```data-toggle="modal"``` doesn't do anything, the confirmation modal is triggered by Javascript. The ```display: inline-block``` doesn't do anything in this case, the whole button group is floated right with ```pull-right```. But we also don't need JS for the edit button, a simple link works just as well, so I've updated the commit with this change. If you want to check it out, it's running at https://mycomics.devel.comix.gr/ (try logging in as adia / testme).